### PR TITLE
August Changelog

### LINKED ISSUE #6574: August Changelog
Re: [Aug 2021 Changelog](https://pantheon.io/docs/changelog)

Priority: Medium

## Issue Description
For platform and documentation updates occurring in July 2021.

### PR DESCRIPTION
**Pull request by joa-pan**
Closes #6574

## Summary

**[Pantheon Changelog](https://pantheon.io/docs/changelog/)** - Updated and added new content for August 2021. Documentation section. Fixed links.

## Effect

The following changes are already committed:

*
*

## Remaining Work

The following changes still need to be completed:

- [ ] List any outstanding work here

--------------------------------------------------
## Post Launch

**Do not remove** - To be completed by the docs team upon merge:

- [ ] Redirect `/docs/old-path/` => `/docs/new-path/` (if applicable)
- [ ] Include/exclude pages ^ respectively within docs search service provider (if applicable)
- [ ] For Heroes - add a props post to the [discussion board](https://discuss.pantheon.io/c/pantheon-platform/documentation/17).
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | [Live](https://pantheon.io/docs) | [pr-6575](https://pr-6575--pantheon-docs.my.pantheonfrontend.website/docs) | Difference |
| --- | --- | --- | --- |
| Performance Score | `53` | `57.99999999999999` | `5` |
| Best Practice Score | `79` | `79` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-6575` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/29059/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/29059/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)


[Preview Environment](https://pr-6575--pantheon-docs.my.pantheonfrontend.website/docs)

File: source/changelogs/2021-08-01.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+### Drupal 9.2.0
+
+[Drupal 9.2.2](https://www.drupal.org/project/drupal/releases/9.2.2) is now available for production sites. [Drupal 9](https://pantheon.io/docs/drupal-9) sites on Pantheon use [Integrated Composer](https://pantheon.io/docs/integrated-composer), enabling one-click core updates through the Dashboard. To check for available updates, click **Check Now** from the site Dashboard **Dev** tab.
+
+Drupal 9.2.x will receive security coverage until June 15, 2022. Sites on 9.1.x or earlier should update to [Drupal 9.1.11](https://www.drupal.org/project/drupal/releases/9.1.11).
+
+
+### Drupal 8.9.17
+
+[Drupal 8.9.17](https://www.drupal.org/project/drupal/releases/8.9.17) is now available on the Pantheon platform. Detailed information on applying and debugging core updates can be found in the [Core Updates documentation](/core-updates).
```
Comment (michellecolon-pantheon) [inline source/changelogs/2021-08-01.md:19]: @joa-pan The Core Updates link is 404'ing here. Not sure why though 🤔

File: source/changelogs/2021-08-01.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+
+### Drupal 7.82
+
+[Drupal 7.82](https://www.drupal.org/project/drupal/releases/7.82) is now available on the Pantheon platform. Pantheon is committed to providing Drupal 7 extended support for all customers through 2025, past Drupal 7’s end of life in 2022.
+
+
+### WordPress 5.8
+
+[WordPress 5.8](https://wordpress.org/news/2021/07/tatum/)[News – Releases – WordPress.org](https://wordpress.org/news/category/releases/) is available on the Pantheon platform. This release fixes security vulnerabilities, and users are urged to upgrade their sites immediately. Detailed information on applying and debugging core updates can be found in the [Core Updates documentation](/core-updates).
```
Comment (michellecolon-pantheon) [inline source/changelogs/2021-08-01.md:29]: Same on this one @joa-pan

**Review by michellecolon-pantheon — COMMENTED** [review]
@joa-pan Please take a look at the comments I left :)

File: source/changelogs/2021-08-01.md (hunk 2 above)
Comment (joa-pan) [reply source/changelogs/2021-08-01.md:29]: i think because the link is relativized. It should resolve when it builds.

**Pull request merged**